### ISSUE TITLE
DOC: clean up DataFrame.ewm

### ISSUE THREAD

**Issue by MarcoGorelli** [pandas-dev/pandas#31647]
A few issues I've noticed:

1) It says

> When adjust is False, weighted averages are calculated recursively as:
> 
>     weighted_average[0] = arg[0]; weighted_average[i] = (1-alpha)*weighted_average[i-1] + alpha*arg[i].
> 

  but `arg` isn't defined

2) The phrase "When adjust is False, weighted averages are calculated recursively as:" appears as bold, but I don't think this was the intention

3) 
```
\text{for} halflife > 0
```
should be
```
\text{for } halflife > 0
```

4) In the "notes" section, ```alpha``` is used, while in the description of the parameters, it's
```
:math:`\alpha` 
```

5) The respective doc for Series could be updated, as it says
```
axis{0 or ‘index’, 1 or ‘columns’}, default 0
```

**Comment by avimiller** [comment]
Me and a few other Data Science Majors I am working with are interested in looking into this as a first contribution if that's okay.

**Comment by MarcoGorelli** [comment]
@avimiller sure, pull requests welcome. Check out the [contributing guide](https://pandas.pydata.org/pandas-docs/stable/development/contributing.html) for how to get started

**Comment by avimiller** [comment]
@MarcoGorelli sounds good, thank you.

**Comment by klakho0400** [comment]
the 3. should be  \text{ for } halflife > 0

**Comment by klakho0400** [comment]
right?

**Comment by klakho0400** [comment]
take

**Comment by klakho0400** [comment]
hey @MarcoGorelli can you pls explain me what can i do for the 1.,5.

**Comment by klakho0400** [comment]
in 5.  does it need to be changed like this  axis{0 or ‘rows', 1 or ‘columns’}, default 0

**Comment by klakho0400** [comment]
in 4.do all alphas in notes need to be replaced by :math:`\alpha`

**Comment by MarcoGorelli** [comment]
1) figure out what the `arg` in the formula refers to, and make it clear

4) the ones in the "notes" section, yes

5) for Series, there is no notion of 'columns', so `axis` would have to refer to only the rows (see other methods such as `set_axis` to see how this differs between the two)

**Comment by MarcoGorelli** [comment]
@avimiller if you're working on this, can you comment 'take' to let others know? Else, it looks like @klakho0400 is also interested

**Comment by mehersaipreetam** [comment]
take

**Comment by MarcoGorelli** [comment]
Closed in #32231
